eo-maven-plugin/src/test/java/org/eolang/maven/OyFake.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import org.cactoos.Func;
+import org.cactoos.Input;
+import org.eolang.maven.objectionary.Objectionary;
+
+/**
+ * Objectionary with lambda-function Ctor-s for testing.
+ *
+ * @since 0.28.11
+ * @checkstyle IllegalCatchCheck (150 lines)
+ */
+public final class OyFake implements Objectionary {
+
+    /**
+     * Function that emulates 'get()' method in {@link Objectionary}.
+     * @checkstyle MemberNameCheck (5 lines)
+     */
+    private final Func<String, Input> lambdaGet;
```

Comment:
@MikhailLipanin we can use `get` and `contains` names. No need in prefix `lambda`. We can understand that from the type of the fields.

Reply:
done

Reply:
@volodya-lombrozo, Can you suggest, how to correctly name these two? We can't just name them as `get` and `contains` since there is already methods `get` and `contains` and Qulice check fails. (since this is a `PMD` error, I can't just mute this errors by `SuppressWarnings` in fields)

Reply:
done